hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py

```diff
@@ -57,12 +120,39 @@ def __init__(self, key: str, value: str):
         super(self.__class__, self).__init__(key, value)
 
 
+class Action:
+    action_label: ActionLabel
+    priority: int
+    superseded_by: t.List[ActionLabel]
+
+    def __init__(self, action_label: ActionLabel, priority: int, superseded_by: t.List[ActionLabel]):
+        self.action_label = action_label
+        self.priority = priority
+        self.superseded_by = superseded_by
+
+
+class ActionRule:
+    action_label: ActionLabel
+    must_have_labels: t.List[Label]
```

Comment:
Wasn't quite thinking about this before but this implies that tags in ThreatExchange at somepoint become Label objects. That seems right to me but where will that take place? In the actioner or the matcher? (or the fetcher?)

Reply:
@jeberl I planned on that being post match, pre-action, and I planned on putting that in to the matcher module based on the beginnings of what @schatten started I think. @schatten what do you think?